libobs/obs-output.h

```diff
@@ -43,7 +43,7 @@ struct obs_output_info {
 
 	void *(*create)(obs_data_t *settings, obs_output_t *output);
 	void (*destroy)(void *data);
-	void (*update_video)(void *user_data, obs_output_t *output);
+	void (*update_av)(void *context, struct obs_video *video, struct obs_sudio *audio);
```

Comment:
In order to maintain compatibility with external plugins, such as Shaderfilter, Shaderfilter-Plus, StreamFX, VirtualCam, and other often used plugins, all changes to a structure like this must be appended to the end of the structure. Elements should not be deleted, but if they are no longer used they should be renamed to an unused tag with the same parameters.

Reply:
I moved update_av to the end of the structure.  update_video was removed.